src/gui/__init__.py

```diff
@@ -14,45 +14,166 @@ def start(self):
+        content_frame.grid(row=0, column=1, sticky="nsew", padx=20, pady=20)
 
-        #Bottom frame: Client Dashboard
-        bottom_frame = ctk.CTkFrame(master=app)
-        bottom_frame.pack(pady=10, padx=10, fill="x")
+        # Search bar
+        search_frame = ctk.CTkFrame(master=content_frame, width=600, height=50)
+        search_frame.pack(fill="x", pady=20)
 
-        #Here we put our client table
-        client_details_textbox = ctk.CTkTextbox(master=bottom_frame, height=500)
-        client_details_textbox.pack(pady=10, fill="x")    
+        search_label = ctk.CTkLabel(master=search_frame, text="Client Dashboard", font=("Arial", 24, "bold"))
+        search_label.pack(anchor="w")
+
+        search_entry = ctk.CTkEntry(master=search_frame, width=400, placeholder_text="Search name or ID...")
+        search_entry.pack(side="left", padx=10, pady=10)
+
+        # Client Table Header
+        table_frame = ctk.CTkFrame(master=content_frame)
+        table_frame.pack(fill="both", expand=True)
+
+        header_frame = ctk.CTkFrame(master=table_frame)
+        header_frame.pack(fill="x", pady=10)
+
+        headers = ["Client ID", "Type", "Name", "Address Line 1", "City", "State", "Zip Code", "Country", "Phone No."]
+        for header in headers:
+            label = ctk.CTkLabel(master=header_frame, text=header, width=10, font=("Arial", 10, "bold"))
+            label.pack(side="left", padx=10)
+
+        # Client Data
+        data_frame = ctk.CTkFrame(master=table_frame)
+        data_frame.pack(fill="both", expand=True)
+
+        # Fetching client data from the repository
+        client_data = self.client_repo.list()
+
+        # Display client data in the UI
+        for client in client_data:
+            row_frame = ctk.CTkFrame(master=data_frame)
+            row_frame.pack(fill="x", pady=5)
+
+            for key in ["ID", "Type", "Name", "Address_Line_1", "City", "State", "Zip_Code", "Country", "Phone_Number"]:
+                value = client.get(key, "")
+                data_label = ctk.CTkLabel(master=row_frame, text=value, width=10)
+                data_label.pack(side="left", padx=10)
+
+        self.app = app
+
+    def open_new_client_window(self):
+        new_window = ctk.CTkToplevel()
+        new_window.title("New Client")
+        new_window.geometry("800x600")
+
+        # Input fields for client information
+        def save_client():
+            new_client = {
+                "Name": name_entry.get(),
+                "Phone_Number": phone_entry.get(),
+                "Address_Line_1": line1_entry.get(),
+                "Address_Line_2": line2_entry.get(),
+                "Address_Line_3": line3_entry.get(),
+                "City": city_entry.get(),
+                "State": state_entry.get(),
+                "Zip_Code": zip_entry.get(),
+                "Country": country_entry.get(),
+                "Type": "Customer"  # Assuming all clients are "Customer"
```

Comment:
I made this mistake before actually. I think this should be "Client" not customer sorry